checks.d/spark.py

```diff
@@ -285,10 +367,44 @@ def _spark_job_metrics(self, running_apps, addl_tags):
                 self._set_metrics_from_json(tags, job, SPARK_JOB_METRICS)
                 self._set_metric('spark.job.count', INCREMENT, 1, tags)
 
+                job_id = job['jobId']
+                previous_status = None
+                if app_id in self.previous_jobs and job_id in self.previous_jobs[app_id]:
+                    previous_status = self.previous_jobs[app_id][job_id]['status']
+                self._event_for_job_status_change(job, tags, previous_status)
+
+            # build index by mapping app ids to a mapping of job id => jobs
+            new_jobs[app_id] = dict((job['jobId'], job) for job in response)
+
+        self.previous_jobs = new_jobs
+
+    def _event_for_job_status_change(self, current_job, tags, previous_status):
+        job_name = current_job['name']
+        job_id = current_job['jobId']
+        current_status = current_job['status']
+        msg_title = "Spark job `%s` is now %s" % (job_name, current_status)
```

Comment:
we should probably favor: `"Spark job {job} is now {status}".format(job=job_name, status=current_status)`